src/stseq.c

```diff
@@ -19,30 +95,32 @@
  * are themselves Table D descriptors are automatically resolved via
  * a recursive call to this same subroutine.
  *
- * @param[in] lun - File ID.
- * @param[in,out] irepct - Replication sequence counter for the current
+ * @param lun - File ID.
+ * @param irepct - Replication sequence counter for the current
```

Comment:
I usually phrase it like this:

`@param irepct - Pointer to the replication sequence counter for the current
`